src/main/java/io/axoniq/axonserver/connector/event/impl/BufferedAggregateEventStream.java

```diff
@@ -31,12 +34,15 @@
         implements AggregateEventStream {
 
     private static final Event TERMINAL_MESSAGE = Event.newBuilder().setAggregateSequenceNumber(-1729).build();
+    private static final int TIMEOUT_MILLIS = Integer.parseInt(System.getProperty("AGGREGATE_TAKE_EVENT_TIMEOUT_MILLIS",
+                                                                                  "10000"));
```

Comment:
Nit: I wouldn't expect the 10000 default to be on a different line honestly. Differently put, indentation is a little off here.

Reply:
I adjusted it as well I could